src/libstd/sys/windows/mod.rs

```diff
@@ -33,11 +33,25 @@ pub mod pipe;
 pub mod process;
 pub mod rand;
 pub mod rwlock;
+#[cfg(not(target_vendor = "uwp"))]
 pub mod stack_overflow;
+#[cfg(target_vendor = "uwp")]
+pub mod stack_overflow_uwp;
+#[cfg(target_vendor = "uwp")]
+pub mod stack_overflow {
+    pub use crate::sys::windows::stack_overflow_uwp::*;
+}
 pub mod thread;
 pub mod thread_local;
 pub mod time;
+#[cfg(not(target_vendor = "uwp"))]
 pub mod stdio;
+#[cfg(target_vendor = "uwp")]
+pub mod stdio_uwp;
```

Comment:
To help avoid some `#[cfg]` in this file, this and the above could also be:

```rust
cfg_if! {
    if #[cfg(target_vendor = "uwp")] {
        mod stdio_uwp;
        mod stack_overflow_uwp;
        pub use self::stdio_uwp as stdio;
        pub use self::stack_overflow_uwp as stack_overflow;
    } else {
        pub mod stdio;
        pub mod stack_overflow;
    }
}
```

Reply:
Done locally, thanks for the hint!